### PR TITLE
Add unmaintained crate informational advisory: rust_sodium

### PR DESCRIPTION
**Pull request by S-Coyle**
rust_sodium is unmaintained, as per [repository readme](https://github.com/maidsafe/rust_sodium#please-note-that-the-rust_sodium-crate-is-no-longer-maintained-switch-to-a-modern-alternative)

### CODE REVIEW

File: crates/rust_sodium/RUSTSEC-0000-0000.toml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+[advisory]
+id = "RUSTSEC-0000-0000"
+
+package = "rust_sodium"
+
+date = "2020-01-20"
+
+title = "rust_sodium in unmaintained; switch to a modern alternative"
```
Comment (brycx) [inline crates/rust_sodium/RUSTSEC-0000-0000.toml]: "in" -> "is"

Comment (S-Coyle) [reply crates/rust_sodium/RUSTSEC-0000-0000.toml]: Well spotted, typo fixed

**Comment by S-Coyle** [comment]
@brycx anything else I need to do for this 1?

**Comment by tarcieri** [comment]
This looks good. Thanks!

**Comment by tarcieri** [comment]
Assigned `RUSTSEC-2020-0003` in #230

**Pull request merged**